docs/modules/automation-testing/pages/start-a-mixed-session.adoc

```diff
@@ -1,7 +1,13 @@
 = Start a mixed session
 :navtitle: Start a mixed session
 
-Learn how to start a mixed session so you can view your Appium script running on your device and fix issues in realtime.
+Learn how to start a mixed session, so you can view your Appium script running on your device and fix issues in realtime.
+
+== What is a mixed session?
+
+A mixed session is where you can use the feature of a manual session to observe what is happening when running an Appium automation script.
```

Comment:
A mixed session allows you to observe what occurs during the execution of an Appium automation script through a manual session.